apps/pxweb2/src/app/app.tsx

```diff
@@ -526,12 +526,13 @@ export function App() {
     <>
       <Header />
       <div className={styles.main}>
-        <div className={styles.desktopNavigation}>
+        <div className={`${styles.desktopNavigation}`}>
```

Comment:
Why this change? I tried changing it back and it still worked?

Reply:
Made suggested change.